ocp_resources/storage_profile.py

```diff
@@ -4,4 +4,33 @@
 
 
 class StorageProfile(Resource):
+    """
+    StorageProfile Object
+
+    Doc:
+    https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md
+    """
+
     api_group = Resource.ApiGroup.CDI_KUBEVIRT_IO
+
+    @property
+    def claim_property_sets(self):
+        return self.instance.status.claimPropertySets
+
+    def first_claim_property_set_access_modes(self):
+        return self.claim_property_sets[0]["accessModes"]
```

Comment:
according to the doc
```
 If the storage provisioner defined in storage class does not have defaults configured in CDI the resulting StorageProfile has empty claimPropertySets.
```
this will fail if empty

Reply:
Yes,
But in cnv-tests it used for StorageProfiles that we expect to have a claimPropertySets defined
So if its missing -> we expect the test to fail

If we just want to check/iterate over a storageprofile CPS, we have claim_property_sets property for it

Reply:
this repository is not tied to cnv-test; if you need some functionality that is not part of the api, please implement there.